tests/test_jurassic_tokenizer.py

```diff
@@ -26,6 +26,21 @@ def test_tokenizer_encode_set(tokenizer: JurassicTokenizer, resources_path: Path
             ), f"Not equal at doc {i}"
 
 
+def test_tokenizer_decode_set(tokenizer: JurassicTokenizer, resources_path: Path):
```

Comment:
`decode_with_offset` should be tested separately from `decode`
You don't have to test `decode` as you didn't add it

Reply:
So simply rename this test and remove the call to `decode`

Reply:
Done